lib/uplinkc/project.go

```diff
@@ -7,48 +7,48 @@ package main
 import "C"
 
 import (
-	"storj.io/storj/lib/uplink"
+	libuplink "storj.io/storj/lib/uplink"
 )
 
 // Project is a scoped uplink.Project
 type Project struct {
 	scope
-	*uplink.Project
+	*libuplink.Project
```

Comment:
Let's keep the `uplink` thing, we'll revert `libuplink` changes in mobile.

Reply:
I think this is better